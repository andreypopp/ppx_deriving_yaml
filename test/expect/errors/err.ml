type t = int -> int [@@deriving yaml]
type u = int -> int [@@deriving yaml]
type v = int [@@deriving yaml]

let _ = v_of_yaml