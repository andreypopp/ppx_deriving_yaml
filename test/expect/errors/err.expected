File "err.ml", line 1, characters 9-19:
Error: Cannot derive anything for this type
